toontown/leveleditor/OTTUtil.py

```diff
@@ -13,3 +13,10 @@ def __task(task):
         return task.cont
 
    return taskMgr.add(__task)
+
+# To safely load the storage files
+def loadStorageFile(pandaPath):
```

Comment:
This file is for all open toontown tools apps so level editor specific stuff shouldn't be here. I'm also pretty sure this function is never actually used

Reply:
submodule when?

Reply:
21